### PR TITLE
Add Letra Extension

### PR DESCRIPTION
**Pull request by vanestorz**
This PR adds Letra, a chrome extension that built using javascript framework both front-end and back-end. It is a good first timers repository since they accepted not only the coding parts, but also translation and testing which is a required skill needed as a developer.

### CODE REVIEW

**Comment by vanestorz** [comment]
@mattjegan could you please review my PR?

**Comment by koppor** [comment]
This is an awesome extension. I love it. Please link only the source code repository. It contains a description, too. Furthermore, this makes the item consistent to the others.

May I ask you to follow our contributing.md?

grafik

May I ask whether our CONTRIBUTING.md is too long? How could we ensure that these requriments are followed?

**Comment by vanestorz** [comment]
> This is an awesome extension. I love it. Please link only the source code repository. It contains a description, too. Furthermore, this makes the item consistent to the others.
> 
> May I ask you to follow our contributing.md?
> 
> grafik
> 
> May I ask whether our CONTRIBUTING.md is too long? How could we ensure that these requriments are followed?

For the description and the link to the official project website i refer to React and React Native section, but i already removed the description and only put the repository link only. For the Contributing.md i truly apologize i misread on some parts and make PR carelessly.

Hope you can re-review my PR and also give a suggestion to make this project better! thanks.

File: readme.md [hunk 1]
```diff
@@ -66,8 +66,10 @@ They also try to be very welcoming for 1st-time contributors.
 ([all JavaScript projects with hacktoberfest issues](https://github.com/search?utf8=%E2%9C%93&q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+language%3AJava&type=Issues&ref=advsearch&l=JavaScript))
 
 - [jest-extended](https://github.com/mattphillips/jest-extended)
+- [letra-extension](https://github.com/jayehernandez/letra-extension)
 - [vue-dropzone](https://github.com/rowanwins/vue-dropzone)
 
+
```
Comment (koppor) [inline readme.md:72]: Why the additional empty line?

File: readme.md [hunk 2]
```diff
@@ -66,8 +66,10 @@ They also try to be very welcoming for 1st-time contributors.
 ([all JavaScript projects with hacktoberfest issues](https://github.com/search?utf8=%E2%9C%93&q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+language%3AJava&type=Issues&ref=advsearch&l=JavaScript))
 
 - [jest-extended](https://github.com/mattphillips/jest-extended)
+- [letra-extension](https://github.com/jayehernandez/letra-extension)
```
Comment (koppor) [inline readme.md:69]: Where is the description gone? In your first proposal, you had a description, which was good as far as I remember.

**Review by koppor — CHANGES_REQUESTED** [review]

**Comment by koppor** [comment]
I fixed it for myself at https://github.com/mattjegan/awesome-hacktoberfest/commit/f240bcc4448c74053dc9efb8f673dd5d3a30861a. Thank you for the inspiration

**Pull request closed without merging**